tests/end_to_end/candid_rpc/class_syntax/new/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "new_end_to_end_test_functional_syntax",
     "scripts": {
-        "test": "if [ \"$AZLE_TEST_RUN_ON_RELEASE\" == \"true\" ]; then tsx test/test.ts; else echo 'Skipping pretests'; fi"
+        "test": "if [ \"$AZLE_TEST_RUN_ON_RELEASE\" = \"true\" ]; then tsx test/test.ts; else echo 'Skipping pretests'; fi"
```

Comment:
Doesn't this break it? If you run that locally doesn't it give an error? Didn't we run into this just a couple weeks ago?

Reply:
yes and we fixed it to a single equal on tests/end_to_end/http_server/new/package.json but we didn't update the class_syntax one